plugin.js

```diff
@@ -245,6 +245,15 @@ Plugin.prototype.finish = function (err, cb) {
   this.q.resume()
 }
 
+// delays plugin loading until the next tick to ensure any bound `_after` callbacks have a chance
+// to run prior to executing the next plugin
+function loadPluginNextTick (toLoad, cb) {
+  const parent = this
+  process.nextTick(() => {
+    loadPlugin.call(parent, toLoad, cb)
+  })
```

Comment:
please avoid creating this function.

Reply:
hey @mcollina - apologies if I'm being dense here, but is the issue here that this work is being delayed until the next tick?  or that it's being done within an anonymous function

Reply:
Just with the anonymous function. Instead of allocating a new closure you can use a top-level function and pass arguments to it:

```js
process.nextTick(console.log, 42)
```

Reply:
oh wow - I had no idea that `process.nextTick()` could accept a list of args 🤦 

I just push a commit with this refactor.  The function still needs to use `.bind(parent)` to set the right context for the `loadPlugin` call (which still creates a new function internally), but the refactor is a bit more readable IMO.